backend/app.js

```diff
@@ -40,7 +40,7 @@ app.get("/api/v2/lumens/total-supply", lumensV2.totalSupplyHandler);
 app.get("/api/v2/lumens/circulating-supply", lumensV2.circulatingSupplyHandler);
 
 app.get("/api/v3/lumens", lumensV3.handler);
-app.get("/api/v3/lumens/total-supply-check", lumensV3.totalSupplySumHandler);
+app.get("/api/v3/lumens/total-supply-check", lumensV3.totalSupplyCheckHandler);
```

Comment:
❗ Are we open to changing the name of this endpoint? `total-supply-check` doesn't really describe the data, since it is more than total supply, and `check` is a verb which makes it sound like calling this endpoint does something.

I suggest `/api/v3/lumens/all`.

Reply:
the purpose of this endpoint is really just for runscope. I only added all the fields so that if it fails runscope will get all the numbers that TotalSupply is calculated from. `all` to me sounds like it's a better version of `/lumens` (atm I guess it is) that people might use instead. I can change to `all` for now, I can't think of a better name for its current use

Reply:
Do we have API documentation for the dashboard?

Reply:
ya  possibly we can combine into a single endpoint. But I think we should make sure this single endpoint check fixes the Runscope error. If not it will have to change anyways